docs/node-operators/how-to/upgrade-network.md

```diff
@@ -161,11 +175,67 @@ In the Vega core logs, you will see the following messages:
 ...
 ```
 
-#### Check if data node is ready for restart
-To see when your data node is ready for protocol upgrade, check the data node logs.
-
-In the data node logs, you will see the following message:
+`(b)`: In the data node logs, you will see the following message:
 
 ```
 2023-03-02T13:01:04.379+0100	INFO	datanode.service	service/protocol_upgrade.go:36	datanode is ready for protocol upgrade
-```
\ No newline at end of file
+```
+
+## 3.3 Restart services
+
+`(a)`: Restart your `core` process
+`(b)`: When you run both: `core` and `data-node`, you must stop both first, and then you can start both in any order.
+
+## 3.4 Automate
+
+Vega Visor is designed to perform `Protocol Upgrade` automatically, but you can automate it yourself. The key is to check if `core` is ready for a restart for `Protocol Upgrade`. As described before, it might take some time (seconds) between the node reaching the `Protocol Upgrade` block and marking itself as ready for the restart. That is why you should not rely on block height, e.g. from `/statistics`.
```

Comment:
```suggestion 
Visor is designed to perform a protocol upgrade automatically, but you can automate it yourself. The key is to check if core is ready for a restart for a protocol upgrade. As described above, it might take several seconds between the node reaches the protocol upgrade block and marks itself as ready to be restarted. Do not solely rely on block height, e.g. from the `/statistics` endpoint.
```